app/build.gradle.kts

```diff
@@ -61,6 +61,7 @@ dependencies {
     implementation(libs.androidx.ui.graphics)
     implementation(libs.androidx.ui.tooling.preview)
     implementation(libs.androidx.material3)
+    implementation(libs.firebase.firestore.ktx)
```

Comment:
I read a [blogpost about Firebase migrating kotlin extension APIs to main modules](https://firebase.google.com/docs/android/kotlin-migration). As a result

> As early as September 2024, we'll no longer release KTX modules

Perhaps we should not use ktx module here. What do you think? 🤔

Reply:
They'll just move the logic when that happens, so the import should automatically be updated when we upgrade the firestore dependency :D

Reply:
Cool 🫘